openeo/NDVI_Timeseries_UDF_Smoothing.ipynb

```diff
@@ -0,0 +1,296 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Timeseries Smoothing\n",
+    "\n",
+    "This notebook is a supplementary notebook to NDVI_Timeseries.ipynb, where we aim to improve the plot's smoothness by using a kernel as an UDF. The kernel will help to reduce noise and fluctuations in the data, resulting in a smoother and more accurate representation of the NDVI timeseries."
```

Comment:
Why starting a whole new notebook with a lot of duplication?
Isn't it cleaner to just continue under the other one with a new section/chapter?

Reply:
combined as one.